expo-maps/android/src/main/java/expo/modules/maps/ExpoMapsModule.kt

```diff
@@ -1,22 +1,40 @@
 package expo.modules.maps
 
+import expo.modules.core.interfaces.LifecycleEventListener
+import expo.modules.core.interfaces.services.UIManager
 import expo.modules.kotlin.modules.Module
 import expo.modules.kotlin.modules.ModuleDefinition
 
 
 class ExpoMapsModule : Module() {
+
+  private val googleMapViews = mutableListOf<GoogleMapsView>()
+
   override fun definition() = ModuleDefinition {
     name("ExpoMaps")
 
-    function("someGreatMethodAsync") { options: Map<String, String> ->
-      println("Hello 👋")
-      null as Any?
-    }
-
     viewManager {
-      view { SolidColorView(it) }
-      prop("color") { view: SolidColorView, color: Int ->
-        view.setColor(color)
+      view {
+        val view = GoogleMapsView(it)
+        googleMapViews.add(view)
```

Comment:
Why do we need this?

Reply:
To call events on all views?

Reply:
Actually googleMapViews list does not bring any value here and could be deleted. It is because registerLifecycleEventListener implementation does not replace previous listeners but instead adds a new listener to its internal map of listeners. Thanks for noticing it Szymon!